apps/docs/src/components/IconPage.astro

```diff
@@ -0,0 +1,21 @@
+---
+import { Icons } from "./Icons/Icons";
+import { parseComponentWithoutAstro } from "@selleo/core";
+
+const { iconsData } = Astro.props;
+
+const iconsList = iconsData.map(({ name, component: Component }: any) => {
+  const { HTMLString, ReactString } = parseComponentWithoutAstro(Component);
+
+  console.log({ HTMLString, ReactString });
```

Comment:
you can remove that line, sorry left it there by accident 😜

Reply:
Alright 🤪